assets/app/scripts/services/metrics.js

```diff
@@ -31,6 +31,11 @@ angular.module("openshiftConsole")
 
     // Convert cumulative CPU usage in nanoseconds to millicores.
     function millicoresUsed(point, lastValue) {
+      // Is there a gap in the data?
+      if (!lastValue || !point.value) {
+        return 0;
+      }
+
       if (lastValue) {
```

Comment:
`if (lastValue && point.value)`  ?  and then drop the new if statement.  you already return 0 at the end of the function